src/client/client4.js

```diff
@@ -207,10 +207,10 @@ export default class Client4 {
 
         if (this.token) {
             headers[HEADER_AUTH] = `${HEADER_BEARER} ${this.token}`;
-        } else {
-            newOptions.credentials = 'include';
         }
 
+        newOptions.credentials = 'include';
```

Comment:
To make sure the cookies are included for the webapp, even if the token is set in the client